app/views/pages/home.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(https://images.unsplash.com/photo-1446057032654-9d8885db76c6?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2000&q=80);">
+<div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(https://res.cloudinary.com/deumrs4dd/image/upload/v1611754380/Sound%20Mate/download_1_yhlk6p.jpg);">
```

Comment:
use the cloudinary method the photo
`https://kitt.lewagon.com/camps/467/lectures/05-Rails%2F05-Rails-MC-with-images`

Reply:
yeah i think vincent changed into unsplash cause he thought javascript didnt work due to it being in cloudinary (which is not the case i think)

Reply:
ok